cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -8688,6 +8688,14 @@ def CreateDatatypes(*ds):
         result.append(dref)
     return tuple(result)
 
+class DatatypeSort:
+    """Unresolved datatype sorts."""
+    
+    def __init__(self, name, ctx=None):
+        self.name = name
+        self.ctx = _get_ctx(ctx)
+        self.ast = self.ctx.tm.mkUnresolvedDatatypeSort(name)
+        
```

Comment:
Instead of creating a new independent class, I would use the `SortRef` class:
```suggestion
def DatatypeSort(name, ctx=None):
    """Create a reference to a sort that will be declared as a recursive datatype"""
    ctx = _get_ctx(ctx)
    s = ctx.tm.mkUnresolvedDatatypeSort(name)
    return SortRef(s, ctx)
```